components/global/product_switcher_menu/product_switcher_menu.tsx

```diff
@@ -68,6 +81,12 @@ class ProductSwitcherMenu extends React.PureComponent<Props> {
 
         return (
             <>
+                <Menu.Group>
+                    <Menu.StartTrial
+                        id='startTrial'
+                        show={this.props.prevTrialLicense?.IsLicensed !== 'true'}
```

Comment:
This should only show for users that have permissions to upload licenses. We should use the same that are used in the System Console > License page.

Reply:
I did this
```jsx
                    <SystemPermissionGate
                        permissions={[Permissions.SYSCONSOLE_WRITE_ABOUT_EDITION_AND_LICENSE]}
                    >
                        <Menu.StartTrial
                            id='startTrial'
                            show={this.props.prevTrialLicense?.IsLicensed !== 'true'}
                        />
                    </SystemPermissionGate>
```